core/modules/utils/dom/dom.js

```diff
@@ -392,3 +392,44 @@ exports.querySelectorAllSafe = function(selector,baseElement) {
 		console.log("Invalid selector: ",selector);
 	}
 };
+
+/*
+Sanitize HTML tag- and custom web component names
+	1. Check the string, if it is a valid html tag - using this spec: https://html.spec.whatwg.org/#syntax-tag-name
```

Comment:
Rather than having this long comment block, couldn't the inline comments be expanded to cover any strictly necessary subtleties?